math/soplex/Portfile

```diff
@@ -24,6 +24,8 @@ checksums           rmd160  30745ef666d7493864fb8e54815c829b9e8bcb4d \
                     sha256  47150ddd6eab9e51003fd2c303ebe8ec225e8ee9742f48548b9c74617d449962 \
                     size    984865
 
+compiler.cxx_standard 2011
```

Comment:
```suggestion
compiler.cxx_standard 2014
```

Reply:
```
/opt/local/libexec/boost/1.76/include/boost/multiprecision/detail/number_base.hpp:36:2: error: "This library now requires a C++11 or later compiler - this message was generated as a result of BOOST_NO_CXX11_HDR_TYPE_TRAITS being set"
```

That lead to thinking that c++2011 is required. But you're right => c++-2014 is required.